pyproject.toml

```diff
@@ -94,6 +94,18 @@ dev = [
     "wheel",
 ]
 
+dev_precommit = [
+    "black",
+    "check-manifest",
+    "flake8",
+    "isort",
+    "nbqa",
+    "nbqa-black",
+    "nbqa-flake8",
```

Comment:
Not sure if it exists before or not, but as of now `nbqa-flake8` does not exist in `PyPI`.